grails-app/views/deliveryNote/_printPage.gsp

```diff
@@ -11,12 +11,12 @@
                     <th><warehouse:message code="packLevel2.label"/></th>
                 </g:if>
                 <th>${warehouse.message(code: 'product.productCode.label')}</th>
-                <th>${warehouse.message(code: 'product.label')}</th>
-                <th>${warehouse.message(code: 'requisitionItem.quantityRequested.label')}</th>
-                <th>${warehouse.message(code: 'requisitionItem.quantityDelivered.label', default: "Delivered")}</th>
+                <th class="left">${warehouse.message(code: 'product.label')}</th>
+                <th>${warehouse.message(code: 'requisitionItem.totalRequested.label', default: "Total Requested")}</th>
+                <th>${warehouse.message(code: 'requisitionItem.totalDelivered.label', default: "Total Delivered")}</th>
                 <th>${warehouse.message(code: 'inventoryItem.lotNumber.label')}</th>
                 <th>${warehouse.message(code: 'inventoryItem.expirationDate.label')}</th>
-                <th>${warehouse.message(code: 'requisitionItem.quantity.label', default: "Quantity")}</th>
+                <th>${warehouse.message(code: 'requisitionItem.deliveredByLoy.label', default: "Delivered by Lot")}</th>
```

Comment:
Typo: `deliveredByLoy` should be `deliveredByLot`. Check messages.properties as well.

Also please change the namespace from requisitionItem to deliveryNote because these since these will probably never be used anywhere other than the deliveryNote. 
```
deliveryNote.totalRequested.label
deliveryNote.totalDelivered.label
deliveryNote.deliveredByLot.label
```